packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -62,6 +63,7 @@ class InnerBlocks extends Component {
 		if ( __experimentalBlocks ) {
 			__unstableMarkNextChangeAsNotPersistent();
 			replaceInnerBlocks( __experimentalBlocks, false );
+			setIsController();
```

Comment:
@youknowriad this is where I'm setting where InnerBlocks is controlled. does this seem like a good place?

Reply:
It seems like the right place yes.

Reply:
maybe do it before the previous two function calls.

Reply:
(yep, that was a good idea so that the reducer knew about the controlled inner blocks before the blocks where there)